README.md

```diff
@@ -2,24 +2,45 @@
 
 This is a monorepo using npm workspaces
 
-### Start client
+### Create local .env file with the following props
 ```
-npm run start -w client
+REDIS_HOST=127.0.0.1
+REDIS_PORT=6379
+GOOGLE_CLOUD_PROJECT='ezla-pickup'
+JWT_SECRET=DEV_SECRET
+FIREBASE_AUTH_EMULATOR_HOST="127.0.0.1:9099"
 ```
 
-### Start backend
+### Run Firebase emulator and Redis locally
+Install Firebase tools:
+```
+curl -sL firebase.tools | bash
 ```
-npm run start -w backend
+Run Firebase auth emulator:
+```
+firebase emulators:start --only auth
 ```
 
-### Start backend mock server
+Run redis-stack docker container locally:
 ```
-npm run start -w backend-mock
+docker run -p 6379:6379 redis/redis-stack:latest
 ```
-This will start a mock server on localhost:8080 with some random data.
 
-Check http://localhost:8080/rides for example
+### Start client
+```
+npm run dev:client
+```
+
+### Start backend
+```
+npm run dev:backend
+```
 
+### Create users for local dev
+RUN GET /dev/init to the local backend api which will create 3 users:
```

Comment:
Can you show the command that needs to be run?
`curl ...` or explain that you simply need to open this url in the browser.